Sources/PointFree/Home.swift

```diff
@@ -53,16 +52,17 @@ let headerView = View<Prelude.Unit> { _ in
           ])
         ]),
       gridColumn(sizes: [:], [
-        div([`class`([Class.grid.end(.mobile)])], [
-          a([href(path(to: .pricing(nil, nil))), `class`([Class.pf.components.button(color: .purple)])], ["Subscribe"])
-          ])
+        div(
+          [`class`([Class.grid.end(.mobile)])],
+          headerLinks.view((currentUser, currentSubscriptionStatus, currentRoute))
+        )
         ])
       ]),
 
     gridRow([`class`([Class.grid.top(.mobile), Class.grid.between(.mobile), Class.padding([.mobile: [.top: 3], .desktop: [.top: 0]])])], [
 
       gridColumn(sizes: [.mobile: 5], [`class`([Class.padding([.mobile: [.top: 4], .desktop: [.top: 0]])]), style(lineHeight(0))], [
-        img(base64: heroLeftMountainSvgBase64, mediaType: .image(.svg), alt: "", [width(.pct(100))])
+        img(base64: heroMountainSvgBase64, mediaType: .image(.svg), alt: "", [width(.pct(100))])
```

Comment:
instead of a left and right mountain svg, we just have a single mountain and the right one is reflected across the x-axis